docs/source/configuration/configuration-gconstruction.rst

```diff
@@ -52,6 +52,7 @@ Similarly, ``edges`` contains a list of edge types and the information of an edg
 
 * ``feature_col``: (**Required**) specifies the column name in the input file that contains the feature.
 * ``feature_name``: specifies the prefix of the column features name. This is optional. If feature_name is not provided, ``feature_col`` is used as the feature name. If the feature transformation generates multiple tensors, ``feature_name`` becomes the prefix of the names of the generated tensors.
+* ``out_dtype`` specifies the data type of the transformed feature. ``out_dtype`` is optional. If it is not set, no data type casting is applied to the transformed feature. If it is set, the output feature will be cast into the corresponding data type Now only flaot16 and float32 are supported.
```

Comment:
Should there be a period between "type" and "Now" in the "... corresponding data type Now only ..."

Reply:
There is a . before Now.
I have updated the sentence.